### DISCUSSION TITLE
What happens when you get banned from GitHub?

### DISCUSSION THREAD

**Discussion by ImmediateMurderProfessionals** [community/community#119815] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Just curious.

**Comment by Niix-Dan** [comment]
When you're banned from GitHub, it typically means that you've violated GitHub's terms of service or community guidelines in some way. The specific consequences of being banned can vary depending on the severity of the violation and GitHub's policies at the time, but common outcomes may include:

**Account Suspension:** Your account may be temporarily or permanently suspended, meaning you won't be able to access any repositories, issues, or other features associated with your GitHub account.

**Loss of Privileges:** You may lose certain privileges such as the ability to create new repositories, fork projects, or interact with issues and pull requests.

**Public Visibility:** Depending on the reason for your ban, GitHub may make your ban public, which can impact your reputation within the developer community.

**Appeal Process:** GitHub typically provides a process for appealing bans. If you believe your ban was unjust or if you've rectified the issue that led to the ban, you can submit an appeal to GitHub.

**Legal Consequences:** In cases of serious violations, there may be legal repercussions, especially if your actions involved copyright infringement, harassment, or other illegal activities.

**Comment by ImmediateMurderProfessionals** [comment]
ok

**Comment by Jbziscool** [comment]
> ### Select Topic Area
> 
> Question
> ### Body
> 
> Just curious.

All your repositories will show the 404 page along with your profile for other people.

When you try to login to github you will get this message.
![image](https://github.com/user-attachments/assets/9f004641-5133-4350-a231-1dc63e8dd676)
